docs/traceflow-guide.md

```diff
@@ -0,0 +1,101 @@
+you can also mark the protocol(TCP/UDP/ICMP) and desired ports of the trace. 
+If you use UI to start a new trace, we currently only supports destination to be Pod, but will support
+destination to be IP or Service soon.
+
+### Using kubectl and YAML file
+You can start a new trace by creating Traceflow CRD via Kubectl and a YAML file which contains the essential
+configuration of Traceflow CRD. An example YAML file of Traceflow CRD might look like this:
+```yaml
+apiVersion: ops.antrea.tanzu.vmware.com/v1alpha1
+kind: Traceflow
+metadata:
+  name: tf-test
+spec:
+  source:
+    namespace: default
+    pod: tcp-sts-0
+  destination:
+    namespace: default
+    pod: tcp-sts-2
+#   ip: IP can also be marked as destination
+  packet:
+    ipHeader:
+      protocol: 6 # Protocol here can be 6 (TCP), 17 (UDP) or 1 (ICMP), default value is 1 (ICMP)
+    transportHeader:
+      tcp:
+        srcPort: 10000 # Source port needs to be set when Protocol is TCP/UDP.
+        dstPort: 80 # Destination port needs to be set when Protocol is TCP/UDP.
+```
+The CRD above starts a new trace from port 10000 of source Pod named tcp-sts-0 to port 80
+of destination Pod named tcp-sts-2 using TCP protocol.
+
+### Using Octant with antrea-octant-plugin
+
+<img src="http://downloads.antrea.io/static/tf_create.png" width="600" alt="Start a New Trace">
+
+From Octant dashboard, you need to click on left navigation bar named "Antrea" and then
+choose category named "Traceflow Info" to lead you to the Traceflow UI displayed on the right side.
+
+Now, you can start a new trace by clicking on the button named "Start New Trace" and submit the form with trace details.
+It helps you create a Traceflow CRD and generate a corresponding Traceflow Graph.
+
+## View Traceflow Result and Graph
+
+You can always view Traceflow result directly via Traceflow CRD status and see if the packet is successfully delivered
+or somehow dropped by certain component. Antrea also provides a more friendly way by showing the Traceflow result via a
+trace graph on UI.
+
+<img src="http://downloads.antrea.io/static/tf_graph_success.png" width="600" alt="Show Successful Trace">
+
+From the graph above, we can see the inter-node traffic between two Pods have been successfully delivered.
+Sometimes the traffic may not be successfully delivered and we can always easily identify where the traffic is dropped
+via a trace graph like below.
+
+<img src="http://downloads.antrea.io/static/tf_graph_failure.png" width="600" alt="Show Failing Trace">
+
+You can also generate historical trace graph by providing certain Traceflow CRD name as shown below.
+
+<img src="http://downloads.antrea.io/static/tf_historical_graph.png" width="600" alt="Generate Historical Trace">
+
+## View Traceflow CRDs
+
+<img src="http://downloads.antrea.io/static/tf_overview.png" width="600" alt="Antrea Overview">
+
+As shown above, you can view the existing Traceflow CRDs from the last table of Antrea Overview on UI.
```

Comment:
what is the "last table"? do you mean the table that comes last in the page layout? I suggest using the name of the table. It is clearer and more future-proof. I suggest this sentence:

> As shown above, you can check existing Traceflow CRDs in the "Traceflow Info" table of the Antrea Overview web page in the Octant UI. You can generate a trace graph for any of these CRDs, as explained in the previous section.